app/src/components/molecules/FloatingIconButton/index.jsx

```diff
@@ -6,11 +6,20 @@ export const FloatingIconButton = ({
     position = 'left',
     iconName,
     onPress,
+    customTop = null,
+    iconSize,
+    color,
 }) => {
     const convertPosition = `${position}-2`;
+    const positionY = customTop ? customTop : 'top-2';
```

Comment:
Incorporando meu amigo Gabriel Davi, é possível fazer isso sem ternário.
```suggestion
    const positionY = customTop || 'top-2';
```

Reply:
feito